js/index.js

```diff
@@ -1,42 +1,110 @@
+    },
+    "contact": {
+        "contact-h4": "Contact",
+        "address": "123 Way 456 Street Somewhere, USA",
+        "phone": "1 (888) 888-8888",
+        "email": "sales@greatidea.io",
+    },
+    "footer": {
+        "copyright": "Copyright Great Idea! 2018"
+    },
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+let ancors = document.querySelectorAll("header nav a");
+ancors.forEach((item, i) => item.textContent = siteContent["nav"][`nav-item-${i}`]);
+
+let ctah1 = document.querySelector(".cta-text h1");
+ctah1.textContent = siteContent["cta"]["h1"];
+
+let ctabtn = document.querySelector(".cta-text button");
+ctabtn.textContent = siteContent["cta"]["button"];
+
+let ctaImg = document.querySelector("#cta-img");
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"]);
+
+let topContentH4 = document.querySelectorAll(".top-content .text-content h4");
+topContentH4[0].textContent = siteContent["main-content"]["features-h4"];
+topContentH4[1].textContent = siteContent["main-content"]["about-h4"];
+
+let topContentP = document.querySelectorAll(".top-content .text-content p");
+topContentP[0].textContent = siteContent["main-content"]["features-content"];
+topContentP[1].textContent = siteContent["main-content"]["about-content"];
+
+let middleImg = document.querySelector("#middle-img");
+middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"]);
+
+let bottomContentH4 = document.querySelectorAll(".bottom-content .text-content h4");
+bottomContentH4[0].textContent = siteContent["main-content"]["services-h4"];
+bottomContentH4[1].textContent = siteContent["main-content"]["product-h4"];
+bottomContentH4[2].textContent = siteContent["main-content"]["vision-h4"];
+
+let bottomContentP = document.querySelectorAll(".bottom-content .text-content p");
+bottomContentP[0].textContent = siteContent["main-content"]["services-content"];
+bottomContentP[1].textContent = siteContent["main-content"]["product-content"];
+bottomContentP[2].textContent = siteContent["main-content"]["vision-content"];
+
+let contactH4 = document.querySelector(".contact h4");
+contactH4.textContent = siteContent["contact"]["contact-h4"];
+
+let contactP = document.querySelectorAll(".contact p");
+contactP[0].textContent = siteContent["contact"]["address"];
+contactP[1].textContent = siteContent["contact"]["phone"];
+contactP[2].textContent = siteContent["contact"]["email"];
+
+let copyright = document.querySelector("footer p");
+copyright.textContent = siteContent["footer"]["copyright"];
+
+//added new content
+ancors.forEach((item) => item.style.color = 'green');
+
+let nav = document.querySelector("header nav");
+let el1 = document.createElement('a');
+let el2 = document.createElement('a');
+el1.textContent = 'Child';
+el2.textContent = 'Child';
+nav.appendChild(el1);
+nav.prepend(el2);
```

Comment:
This is shorter and more efficient than many that I've seen while grading these, so that's awesome.